src/masternode-payments.cpp

```diff
@@ -675,57 +664,35 @@ bool CMasternodePayments::ProcessBlock(int nBlockHeight)
 {
     if(!fMasterNode) return false;
 
-    //reference node - hybrid mode
-
-    if(!IsReferenceNode(activeMasternode.vin)){
```

Comment:
Yeah, we definitely need that logic, but without the reference node. Fixed in ff06e58186ab39d40f7ba840e158433fcf1bc2a0